smoketest/scripts/cli/test_vpn_ipsec.py

```diff
@@ -282,6 +282,80 @@ def test_site_to_site(self):
         for line in swanctl_secrets_lines:
             self.assertRegex(swanctl_conf, fr'{line}')
 
+    def test_site_to_site_ts_protocol_all(self):
+        """
+        Test acceptance of 'all' protocol in site-to-site traffic selector.
+
+        Verifies that specifying only the subnet (e.g., 'x.x.x.0/24') is accepted
+        for "all" protocols in IPsec site-to-site configuration, while explicit
+        '[all/]' protocol syntax is rejected with strongSwan 5.9.x.
+
+        More details: https://vyos.dev/T7581
+        """
+
+        self.cli_set(base_path + ['ike-group', ike_group, 'key-exchange', 'ikev2'])
+
+        local_address = '192.0.2.12'
+
+        # vpn ipsec auth psk <tag> id <x.x.x.x>
+        self.cli_set(
```

Comment:
Those paths don't seem long enough to me to require a line break here. It's a purely subjective thing, of course, not an obstacle for merging.

Reply:
Sometime it is difficult to balance between rules of the linter and readability. I've pushed updated version with reduced lines.